src/runtime_src/core/pcie/linux/system_linux.cpp

```diff
@@ -279,3 +335,25 @@ get_device_id_from_bdf(const std::string& bdf)
 } // pcie_linux
 
 } // xrt_core
+
+namespace pcidev {
+
+size_t
+get_dev_ready(bool user)
+{
+  return singleton_instance()->get_num_dev_ready(user);
```

Comment:
`singleton_instance()` is not an informative name.

Reply:
Changed to "singleton_system_linux"